src/Components/ArbitrageDashboardTable/ArbitrageDashboardTable.js

```diff
@@ -28,200 +23,22 @@ const useStyles = makeStyles({
-      price_difference: 0.00006913613
-    },
-    {
-      buy_exchange: "bitfinex",
-      sell_exchange: "coinbase_pro",
-      trading_pair: "btc_eth",
-      arbitrage_percentage: 0.05,
-      trade_time: 1576537496,
-      price_difference: 0.0001698136
-    },
-    {
-      buy_exchange: "binance",
-      sell_exchange: "coinbase_pro",
-      trading_pair: "eth_ltc",
-      arbitrage_percentage: 0.03,
-      trade_time: 1576537496,
-      price_difference: 0.00006369813
-    },
-    {
-      buy_exchange: "coinbase_pro",
-      sell_exchange: "bitfinex",
-      trading_pair: "matic_xmr",
-      arbitrage_percentage: 0.02,
-      trade_time: 1576537496,
-      price_difference: 0.000236913
-    },
-    {
-      buy_exchange: "binance",
-      sell_exchange: "bitfinex",
-      trading_pair: "dash_ltc",
-      arbitrage_percentage: 0.02,
-      trade_time: 1576537496,
-      price_difference: 0.0003691321
-    },
-    {
-      buy_exchange: "binance",
-      sell_exchange: "bitfinex",
-      trading_pair: "btc_dash",
-      arbitrage_percentage: 0.07,
-      trade_time: 1576537496,
-      price_difference: 0.000631362114
-    },
-    {
-      buy_exchange: "coinbase_pro",
-      sell_exchange: "binance",
-      trading_pair: "ltc_usd",
-      arbitrage_percentage: 0.03,
-      trade_time: 1576537496,
-      price_difference: 0.00036124211
-    },
-    {
-      buy_exchange: "bitfinex",
-      sell_exchange: "coinbase_pro",
-      trading_pair: "btc_xrp",
-      arbitrage_percentage: 0.06,
-      trade_time: 1576537496,
-      price_difference: 0.000062136219
-    },
-    {
-      buy_exchange: "binance",
-      sell_exchange: "coinbase_pro",
-      trading_pair: "ltc_btc",
-      arbitrage_percentage: 0.05,
-      trade_time: 1576537496,
-      price_difference: 0.0003611362114
-    },
-    {
-      buy_exchange: "coinbase_pro",
-      sell_exchange: "bitfinex",
-      trading_pair: "btc_usd",
-      arbitrage_percentage: 0.04,
-      trade_time: 1576537496,
-      price_difference: 0.000813621485
-    },
-    {
-      buy_exchange: "bitfinex",
-      sell_exchange: "binance",
-      trading_pair: "eth_usd",
-      arbitrage_percentage: 0.02,
-      trade_time: 1576537496,
-      price_difference: 0.0005322146958
-    },
-    {
-      buy_exchange: "binance",
-      sell_exchange: "bitfinex",
-      trading_pair: "xrp_usd",
-      arbitrage_percentage: 0.03,
-      trade_time: 1576537496,
-      price_difference: 0.0000813662164
-    },
-    {
-      buy_exchange: "bitfinex",
-      sell_exchange: "coinbase_pro",
-      trading_pair: "btc_usd",
-      arbitrage_percentage: 0.07,
-      trade_time: 1576537496,
-      price_difference: 0.00085226614988
-    }
-  ];
+
+  const [data, setData] = useState([]);
+ 
+
+useEffect(() => {
+  
+axios.get("https://cryptolytic-starter.herokuapp.com/arbitrage")
+     .then(res => { 
+      setData(res.data);
+         
+     })
+     .catch(err => console.log(err.response));
+
+}, []);
+
+
```

Comment:
Went over with Jeff and was able to advise and demonstrate that back end was built out and live responding to axios calls.